From f53098a82a9159a880965abb23b31ee827635049 Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Wed, 4 Nov 2015 08:35:20 +0200
Subject: [PATCH 0461/1240] fix: ddr: Set DDR size to 4GB, until 64bit DDR
 issue is resolved

	A work-around until 64-bit DDR issue is resolved. No need to use "mem="
	for Linux boot.

Change-Id: I085f027d39555a16ed80488f62e4d96ad0e8db9d
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/24563
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 drivers/ddr/ddr_mckinley6.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/ddr/ddr_mckinley6.c b/drivers/ddr/ddr_mckinley6.c
index 0c69432..502d768 100644
--- a/drivers/ddr/ddr_mckinley6.c
+++ b/drivers/ddr/ddr_mckinley6.c
@@ -897,7 +897,7 @@ struct mvebu_mckinley_config mckinley_phy_config[] = {
 	{ 0x8c0, { 0xc1000000} },
 	{ 0x98 , { 0x0040004e} },
 	{ 0x108, { 0xffff0002} },
-	{ 0x200, { 0x00110001} },
+	{ 0x200, { 0x00100001} },
 	{ 0x204, { 0x00000000} },
 	{ 0x220, { 0x11010642} },
 	{ 0x44 , { 0x00030300} },
-- 
1.9.1

